### DISCUSSION TITLE
Using splash screen without logo

### DISCUSSION THREAD

**Discussion by kyratolnai11** [zoontek/react-native-bootsplash#607] [General]
Is there a way to use the splash screen without a logo? I still want to have a splash screen while my complex animation loads on a separate screen, but I do not want it to contain a logo. Right now I use a transparent PNG to generate the assets and overcome the issue, but it would be nice to have a better way 😄

**Comment by zoontek** [comment]
The `logo` property is already not mandatory: https://github.com/zoontek/react-native-bootsplash/blob/2572a8d2fa5aeab85c6667dceb023e4ae14c5dce/src/index.ts#L35

**Comment by kyratolnai11** [comment]
but if i try to run the command without adding a logo, I am getting an error saying missing required argument logo. The command I am trying to run is: 

yarn react-native generate-bootsplash  \  
  --platforms=ios \
  --background=#0045b7

**Reply by zoontek** [reply]
Yes, pass any image, just don't use it. Remove the bootSplashLogo in the styles.xml, remove it from the storyboard, and don't pass it to the hook.

**Reply by ronickg** [reply]
> Yes, pass any image, just don't use it. Remove the bootSplashLogo in the styles.xml, remove it from the storyboard, and don't pass it to the hook.

Would it be possible to add it to the expo settings as well, so one doesn't need to change it after each prebuild again. So for expo something like this: 

```
  android?: {
    parentTheme?: "TransparentStatus" | "EdgeToEdge";
    darkContentBarsStyle?: boolean;
    logo: false,
  };
  ios?: {
    logo: false,
  }
  ```
